### PR TITLE
Update deprecated-images.yml

### PR DESCRIPTION
**Pull request by Olosinsk-commits**
based on the workitem https://msazure.visualstudio.com/One/_workitems/edit/29745968

### CODE REVIEW

**Comment by Olosinsk-commits** [comment]
#assign-reviewer: ebolton-cyber

**Comment by Olosinsk-commits** [comment]
#sign-off (public repo)

**Comment by ShannonLeavitt** [comment]
@ju-shim

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by ebolton-cyber — APPROVED** [review]
Changes look good. This will optimize the cli commands/scripts for customers to find deprecated images more easily

**Review by ebolton-cyber — APPROVED** [review]

File: articles/virtual-machines/deprecated-images.yml [hunk 1]
```diff
@@ -96,25 +99,27 @@ sections:
...
           
-          List scale sets with deprecated images at plan/SKU level.
+          - List scale sets with deprecated images at plan/SKU level.
           
           ```powershell
           $vmsslist = Get-AzVmss
           $vmsslist | where {$_.virtualMachineProfile.storageProfile.imageReference.Sku -eq '2016-Datacenter'} | Select-Object -Property ResourceGroupName, Name, @{label='imageOffer'; expression={$_.virtualMachineProfile.storageProfile.imageReference.Offer}}, @{label='imagePublisher'; expression={$_.virtualMachineProfile.storageProfile.imageReference.Publisher}}, @{label='imageSKU'; expression={$_.virtualMachineProfile.storageProfile.imageReference.Sku}}, @{label='imageVersion'; expression={$_.virtualMachineProfile.storageProfile.imageReference.Version}}
           ```
+
+          - Run [Get-AzVMImageDeprecationStatus.ps1](https://github.com/Azure/azure-support-scripts/tree/master/VMImages#readme) to **list if any VMs or VMSS in a subscription were created from VM images that are scheduled for deprecation**.
```
Comment (ju-shim) [inline articles/virtual-machines/deprecated-images.yml:122]:
```suggestion
          - Run [Get-AzVMImageDeprecationStatus.ps1](https://github.com/Azure/azure-support-scripts/tree/master/VMImages#readme) to list if any VMs or VM scale sets in a subscription were created from VM images that are scheduled for deprecation.
```

**Review by ju-shim — CHANGES_REQUESTED** [review]

**Review by ju-shim — APPROVED** [review]

**Comment by ju-shim** [comment]
#sign-off

**Pull request merged**